go/integration_test/processor/world_state_test.go

```diff
@@ -17,6 +17,7 @@ import (
 	"testing"
 
 	"github.com/Fantom-foundation/Tosca/go/tosca"
+	"github.com/Fantom-foundation/Tosca/go/tosca/vm"
 
 	// This is only imported to get the EVM opcode definitions.
 	// TODO: write up our own op-code definition and remove this dependency.
```

Comment:
Since this import is no longer required it should be removed as well as the comment. Please use your IDE search function to find all other files where this is applicable.

Reply:
nice catch !!